projects/components/src/file-upload/file-upload.component.ts

```diff
@@ -46,6 +46,13 @@ import { FileTypeUtil, SupportedFileType } from './file-types';
 
     <ng-template #defaultSubTpl>
       <ul class="file-requirements-list">
+        <li class="item" *ngFor="let req of (this.config?.customRequirementsInfo ?? [])">
```

Comment:
what's the expected list size? just to double check that it doesn't break anything when the list in long.

Reply:
Ideally, 1 or 2.